hypertrace-trace-enricher/hypertrace-trace-enricher/src/main/resources/configs/common/application.conf

```diff
@@ -107,7 +107,21 @@ enricher {
   }
 
   SpaceEnricher {
-    class = "org.hypertrace.traceenricher.enrichment.enrichers.SpaceEnricher"
+    class = "org.hypertrace.traceenricher.enrichment.enrichers.space.SpaceEnricher"
+
+    config.service.config = {
+      host = localhost
+      host = ${?CONFIG_SERVICE_HOST_CONFIG}
```

Comment:
Do we need the `_CONFIG` suffix?

Reply:
Consistency with the other variables: 
![image](https://user-images.githubusercontent.com/45047841/104852100-9f4ad480-58ad-11eb-924e-d28bf95d6a2c.png)